### ISSUE TITLE
Rats are stuck holding cheese and won't do anything (1.20.1)

### ISSUE THREAD

**Issue by Pineappleguy03** [Alex-the-666/Rats#838]
After taming a rat. The rat refuses to do anything else and somehow has cheese in their hand when nothing is in the food slot. Any upgrades or anything does nothing at all.

**Comment by Pineappleguy03** [comment]
Update: After Resetting the server they have the clothes but visually do not update if wearing an armor set or something. Only Resetting the server causes them to update.

**Comment by slickritalin** [comment]
Having the same issue

**Comment by GizmoTheMoonPig** [comment]
Is this in the DireWolf20 1.20 modpack? Ive been informed of similar things happening in that pack from other people and I have no idea whats causing it. I havent had a chance to really look into it as ive been away but I'll be able to soon

**Comment by Pineappleguy03** [comment]
No! This is my own modpack! But hopefully you will be able to fix it! I think it's something with Canary.

**Comment by GizmoTheMoonPig** [comment]
turns out this is indeed a canary issue! I need to see if theres some sort of setting that needs to be changed to fix this, but I doubt I can work around it in the mod itself

**Comment by Pineappleguy03** [comment]
Did we ever fix this?

**Comment by meglazero** [comment]
Still appears to be an issue with canary, modpack I was playing has Radium Reforged installed which appears to use/contain canary and rats will not update items given/taken without reloading the world

**Comment by Pineappleguy03** [comment]
Did you find a workaround?

**Comment by Misticfury** [comment]
I had a similar issue but with the classes, yours might be similar but with the AI. go into the canary config file a turn off the optimization that's causing you issues. :)
https://github.com/AbdElAziz333/Canary/wiki/Configuration-File

**Comment by Pineappleguy03** [comment]
Can you share an example?

**Comment by Pineappleguy03** [comment]
Because I dunno what it is....

**Comment by Pineappleguy03** [comment]
UPDATE: So. I did some changes to disable everything related to AI in canary and they just will not work. I think this is just a bug. But hopefully this gets fixed soon.

**Comment by johalun** [comment]
I'm having a similar problem which could be the same bug. My fisherman rat will fish but not deposit the item into the chest, so it will just keep on fishing but never get any additional items since there's already something in it's inventory. However another rat that just harvests crops works normally. 

1.20.1
Own modpack with ~250 mods

**Comment by johalun** [comment]
Same with a rat with no upgrades set to transport items. They will pick up an item, then just sit in place and do nothing.

**Comment by Pineappleguy03** [comment]
Unfortunately you are right. If you also set a rat to patrol it will not do anything. Seems to be the AI or something because ice and fire, including rats seems to be bugging out. I can't seem to fix this either.

**Comment by johalun** [comment]
Do you know if this issue has always existed on 1.20.1? I wonder if downgrading could fix it...

**Comment by johalun** [comment]
Found the issue.. It has to do with claims (using Cadmus). Can't find any claim setting that will allow the rat to work though..
